client/ayon_slack/plugins/publish/integrate_slack_api.py

```diff
@@ -49,32 +57,26 @@ def process(self, instance):
             project = instance.context.data["anatomyData"]["project"]["code"]
             for channel in message_profile["channels"]:
                 if six.PY2:
```

Comment:
It is question out of this PR, but why is `client` created for each message profile and each channel? Wouldn't be easier to create `client` only once at the top?